src/app/not-found.tsx

```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link';
+
+const NotFound = () => (
```

Comment:
How does custom 404 work in app router? Because it seems it should still be achieved via the same 404 page in pages: https://nextjs.org/docs/pages/building-your-application/routing/custom-error

Reply:
app/not-found.tsx seems to be the convention https://nextjs.org/docs/app/api-reference/file-conventions/not-found 😊